README.md

```diff
@@ -81,7 +81,7 @@ interface IAuthContext {
   tokenData?: TTokenData
   // Function to trigger login. 
   // If you want to use 'state', you might want to set 'clearURL' configuration parameter to 'false'.
-  login: (state?: string) => void  
+  login: (state?: string, additionalParameters?: { [key: string]: string | boolean | number }) => void  
```

Comment:
Considering how this type is repeated a lot of places, maybe it should be defined somewhere else, and just reused? DRY and all that.